content/webapp/services/wellcome/common/filters.ts

```diff
@@ -121,9 +136,13 @@ function filterOptionsWithNonAggregates({
     )
     .filter(option => isNotUndefined(option)) as FilterOption[];
 
-  return allOptions
-    .filter(option => showEmptyBuckets || option.count || option.selected)
-    .sort(optionOrder);
+  if (!isManualSort) {
+    return allOptions
+      .filter(option => showEmptyBuckets || option.count || option.selected)
+      .sort(optionOrder);
+  }
+
+  return allOptions;
```

Comment:
We don't want it filtered out (if there are no "future" events for example, it doesn't go away, it just becomes disabled).